src/common/cd-special-req-component.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+<div>
+  <select class="select-box form-control" v-model="specialReqSelect" @blur="onBlur" @focus="onFocus">
+    <option value="" selected data-default>{{ $t('No') }}</option>
+    <option value="yes">{{ $t('Yes') }}</option>
+  </select>
+  <input v-show="hasSpecialReq" class="form-control" v-model="specialReqInput" :placeholder="$t('Please type any special requirements you have here (ex. Need wheelchair access)')" @blur="onBlur" @focus="onFocus"/>
+</div> 
+</template>
+
+<script>
+  export default {
+    name: 'cd-special-req-component',
+    props: ['value'],
+    data() {
+      return {
+        specialReqSelect: '',
+        specialReqInput: '',
+      };
+    },
+    methods: {
+      onBlur() {
+        this.blurTimeout = window.setTimeout(() => {
+          this.$emit('blur');
+        }, 50);
+      },
+      onFocus() {
+        window.clearTimeout(this.blurTimeout);
+      },
+    },
+    computed: {
+      hasSpecialReq() {
+        return this.specialReqSelect === 'yes';
+      },
+      specialRequirement: {
+        get() {
+          if (this.hasSpecialReq) {
+            return this.specialReqInput;
+          }
+          return this.specialReqSelect;
+        },
+        set(specialReqVal) {
+          if (!specialReqVal) {
+            this.specialReqSelect = 'yes';
```

Comment:
shouldn't that be the opposite?
When specialReq is set, it's a "yes" and the input is filled.
When specialReq is not set, it's a no, which means resetting the input to null and resetting the select to empty

Reply:
yes you are correct my bad